### ISSUE TITLE
Support macaddr data type

### ISSUE THREAD

**Issue by ksaitor** [balderdashy/sails-postgresql#138]
I've got a `mac_address macaddr NOT NULL UNIQUE` in my PG database.
And in my model using `mac_address: { type: 'string', unique: true, required: true }`
In production getting this error
```
Unhandled rejection [Error (E_UNKNOWN) Encountered an unexpected error] Details:  error: function lower(macaddr) does not exist
```

**Comment by sailsbot** [comment]
Thanks for posting, @ksaitor.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
